### DISCUSSION TITLE
Cannot change user's password using XAMPP localhost

### DISCUSSION THREAD

**Discussion by kimku99** [datenstrom/community#580] [Report a bug] [answered]
Hello, 
I try to change user's password from localhost (XAMPP).
I changed the php.ini and sendmail.ini to send emails from Gmail.
I can receive the email title as "Change user account" and the contain:
```
please confirm that you want to change your user account. Click the following link.

http://localhost/edit/action:change/email:USER@gmail.com/expire:
1608701222/language:en/actiontoken:bfb......
```
when I click the link, I got "Page not found":
```
Page not found
The requested page was not found. You can create this page.
```
when I click the link "You can create this page", I got:
```
---
Title: Actiontoken
---
```

It seems it doesn't recognize `Actiontoken`

**Comment by kimku99** [comment]
Oh no! This is a bug!
After updating to the new version of Edit extension, the problem is solved

**Comment by markseuffert** [comment, marked as answer]
This was a Windows related problem in the [edit extension](https://github.com/annaesvensson/yellow-edit). When you change a user account in the web browser the server sends a confirmation email with link. The generated link was wrong. Very likely it was wrong from the beginning and never tested. Location arguments have to use a different separator based on the operating system.

For example the link looks like `http://localhost/edit/action:change/more:data/`  
but on Windows it should be `http://localhost/edit/action=change/more=data/`.

Should be fixed now.
